services/datamanager/builtin/builtin.go

```diff
@@ -686,12 +686,16 @@ func (svc *builtIn) sync() {
 	svc.lock.Unlock()
 
 	stopAfter := time.Now().Add(time.Duration(svc.syncIntervalMins * float64(time.Minute)))
+	// Logging to display how backed up data sync is
+	if len(toSync) >= 1000 {
```

Comment:
Sorry for changing this number again but lets change this to only log >5000, and can you update the comment to include something like `log only if there are a large number of files to sync` basically just saying we only want to log this if theres a lot of capture files to avoid flooding a users robot logs.

Reply:
Resolved!